rolling/distribution.yaml

```diff
@@ -5798,8 +5798,8 @@ repositories:
     release:
       tags:
         release: release/rolling/{package}/{version}
-      url: https://github.com/ros2-gbp/septentrio_gnss_driver_ros2-release.git
-      version: 1.2.3-2
+      url: https://github.com/septentrio-users/septentrio_gnss_driver_ros2-release
```

Comment:
This is not allowed in Rolling, because otherwise our automated scripts for branching off the distribution will not work.

Reply:
(that is, you should just keep using the `ros2-gbp` release repository)